src/AzureRMTools.ts

```diff
@@ -56,7 +56,7 @@ export class AzureRMTools {
 
     private productionTelemetryProperties: Telemetry.Properties = {
         productName: "azurerm-vscode-tools", // Keep this value synchronized with the name in package.json
-        productVersion: "0.3.4", // Keep this value synchronized with the version in package.json
+        productVersion: "0.3.8", // Keep this value synchronized with the version in package.json
```

Comment:
Doesn't this extension use the 'vscode-extension-telemetry' package? These seem to overlap with the common properties: https://github.com/Microsoft/vscode-extension-telemetry#common-properties

(And obviously someone didn't keep this version in sync lol)